tests/functional/intelligent-host-selection/03-polling.t

```diff
@@ -36,15 +42,15 @@ create_test_global_config "" "
         install target = ${CYLC_TEST_INSTALL_TARGET}
         retrieve job logs = True
         communication method = poll
-        execution polling intervals = PT0S, PT14M
+        execution polling intervals = 10*PT3S
```

Comment:
Want to wait a decent interval (3x the bad-host clearance time seems ok), then repeat a fair number of times.